db/migrate/20190305112030_add_state_indexes_to_listings.rb

```diff
@@ -0,0 +1,6 @@
+class AddStateIndexesToListings < ActiveRecord::Migration[5.1]
+  def change
+    add_index :listings, [:community_id, :open, :state, :deleted, :valid_until, :sort_date], name: 'listings_homepage_query'
+    add_index :listings, [:community_id, :open, :state, :deleted, :valid_until, :updates_email_at, :created_at], name: 'listings_updates_email'
```

Comment:
@bladealslayer 
Literary implemented. EXPLAIN does not show improvement for me. Not merged in staging due to hard rollback.

Reply:
@aivils the old `homepage_query` and `updates_email_listings` can be removed, no need to keep them around. Just make sure they are removed after the new ones are created.

Reply:
@aivils did you see some need to add `:created_at` to the second index? I'm not saying it shouldn't be there, I just didn't notice that it was needed.

Reply:
@bladealslayer 
Both updates_email_at and created_at are used.
https://github.com/sharetribe/sharetribe/blob/master/app/models/community.rb#L543

Reply:
Ok, so as soon as you add code to remove the old unnecessary indexes, this PR LGTM @aivils